### ISSUE TITLE
Add a quick function to notifications, like Mark As Read

### ISSUE THREAD

**Issue by Ortieez** [AniTrend/anitrend-app#417]

**Comment by taichikuji** [comment]
# AniTrend Issue Guidelines

Before opening a new issue, please take a moment to review our [**community guidelines**](https://github.com/AniTrend/anitrend-app/blob/master/CONTRIBUTING.md) to make the contribution process easy and effective for everyone involved.

**You may find an answer in already closed issues**:
https://github.com/AniTrend/anitrend-app/issues?q=is%3Aissue+is%3Aclosed


## Feature Information 


Suggestion to add a similar feature, related usually with Social Media applications, to "mark as read" on a notification.

Example below:
![Mark As Read Picture](https://i.imgur.com/RxXtchZ.png "Mark As Read Example")


## Solution Information


A clear and concise description of what you want to happen.

I think something like what I've shared would be what we want. The current option (picture below) is "good enough" as it does its job, but it's too unintuitive to enable and feels more like a workaround rather than a feature.

Example:
![Slide to dismiss](https://cdn.discordapp.com/attachments/314444307278856192/831768631452631060/IMG_20210414_075122.jpg "Current version of the feature")

**Describe alternatives you've considered**
A clear and concise description of any alternative solutions or features you've considered.

Alternatives would be the current "Dismiss to clear notifications", but enabled by default to everyone and only being disabled manually by the user, or considering adding something in between where you could have both the "Mark as Read" button, plus the Dismiss to clear notifications.

## Additional context
Add any other context or screenshots about the feature request here.

Personally, I think that the best approach would be Adding the "Mark as Read" button, and removing the Dismiss to clear notifications feature from the settings. That would make people have to know that they actually mark as read instead of being automated in a way that maybe some people could feel like it's out of their control.

I leave suggestions and other options open since my approach is could easily not be the best one.

**Comment by taichikuji** [comment]
I'm guessing this can be closed now?

**Comment by luk1337** [comment]
It should close itself automatically once develop branch is merged into master.
